core/storage/suggestion/gae_models.py

```diff
@@ -297,6 +297,27 @@ def query_suggestions(cls, query_fields_and_values):
 
         return query.fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_translation_suggestions_in_review_by_exp_id(cls, exp_id):
```

Comment:
Maybe ``get_translation_suggestions_in_review_with_exp_id`` would be clearer? 'by' seems like the wrong word--the exploration ID isn't conducting the review. :) This also  has better parity with the function name below.

Reply:
Done!